package.json

```diff
@@ -20,35 +20,35 @@
     "clean-install": "rm -rf node_modules && npm cache clean --force && watchman watch-del-all && yarn"
   },
   "dependencies": {
-    "@expo/vector-icons": "^12.0.0",
+    "@expo/vector-icons": "^12.0.5",
     "@react-native-community/masked-view": "0.1.10",
     "@react-navigation/drawer": "^5.11.4",
     "@react-navigation/native": "^5.8.10",
     "@react-navigation/stack": "^5.12.8",
-    "expo": "^41.0.0",
-    "expo-app-loading": "^1.0.1",
-    "expo-asset": "~8.3.1",
+    "expo": "^41.0.1",
+    "expo-app-loading": "^1.0.3",
+    "expo-asset": "~8.3.2",
     "expo-font": "~9.1.0",
     "expo-linear-gradient": "~9.1.0",
-    "lodash": "^4.17.20",
+    "lodash": "^4.17.21",
     "prop-types": "^15.7.2",
     "react": "16.13.1",
     "react-dom": "16.13.1",
     "react-native": "https://github.com/expo/react-native/archive/sdk-41.0.0.tar.gz",
-    "react-native-appearance": "~0.3.3",
-    "react-native-elements": "https://github.com/react-native-elements/react-native-elements#dist",
-    "react-native-elements-universe": "https://github.com/react-native-elements/react-native-elements-universe#dist",
```

Comment:
These are here for a reason, the `next` pre-release version

Reply:
Urm, I saw it was 'next', but it also your default branch, thus defacto 'main'/'master' branch.  Strange you render your default branch useless.